### ISSUE TITLE
Lodash deprecation warning

### ISSUE THREAD

**Issue by NullDivision** [cowboy/node-globule#21]
It would appear that lodash < 3.0.0 now throws a deprecation notice. Might consider upgrading to the latest version considering the functions are standard across versions.

**Comment by abdonrd** [comment]
+1

**Comment by jdillworth** [comment]
+1

**Comment by jdalton** [comment]
Are you still seeing deprecation warnings for lodash?

**Comment by NullDivision** [comment]
Hello Mr. Dalton. Huge fan. :grinning: 

Now that you mention it, I do not. Thank you. That was an unlikely solution :)) Either way, I think updating would be good so I'll keep the issue open.

**Comment by guirong** [comment]
+1

**Comment by bradchesney79** [comment]
node-sass asks for gaze 0.5.1 --> 0.5.2 which asks for globule 0.1.0-- making a pull request to gaze (update gaze package.json to depend on globule 0.2.0) if testing goes well, might lessen the misplaced ire...

**Comment by NullDivision** [comment]
I'm seeing the deprecation warnings again. @jdalton Didn't you remove the deprecation WARN on lodash < 3.0.0?

If no one's gonna update, I will.

**Comment by jdalton** [comment]
> Didn't you remove the deprecation WARN on lodash < 3.0.0?

I reduced the warnings to just those using < 2.0.0.

**Comment by NullDivision** [comment]
Apparently the culprit is `grunt-legacy-util`. Someone already beat me to the punch with a pull request. gruntjs/grunt-legacy-util#9

Given that it's Grunt itself, I'm not putting too much hope into how long it'll take to merge it. I don't even know how it got that outdated.

**Comment by bradchesney79** [comment]
Will have to update node-sass dependency for newer gaze, gaze dev doesn't want to minor bump globule because of major bump on lodash (the whole reason for me digging in and looking around). So, on to plan B.

**Comment by NullDivision** [comment]
@bradchesney79 Currently I say the bigger issue is `grunt-legacy-util` which requires lodash 0.9.x.

**Comment by bradchesney79** [comment]
@NullDivision , completely true for grunt users. gulp-sass->node-sass->gaze->globule->lodash is also generating warnings from old lodash. Just chiming in on elsewhere upstream that things are going sideways for this same issue. I am not positive that gulp users are having the grunt-legacy-util issues.

I would wager looking more closely at the wrapper libraries in the list compiled on the node-sass github would yield a few old dependencies lollygaggers in addition to ours.

I pull requested my minor bump of gaze for node-sass to the wrong repo/branch this morning. But, pull request #2 ready to fire off. It worries me that node-sass only installs correctly for me on node v0.10.x (v0.10.41 was successful specifically). TravisCI only passes for the same v0.10.x for me. Your TravisCI runs pass from top to bottom...

You mentioned one library, by name, not getting a bump because of a breaking change-- that library wasn't gaze. Any particular reason gaze didn't get a bump?

**Comment by travellingprog** [comment]
@bradchesney79 @NullDivision I can confirm that gulp users have no dependency on grunt-legacy-util. However, using gulp itself does generate that warning. Here's the dependency tree I got after running `npm ls lodash@1`:

```
└─┬ gulp@3.9.0
  └─┬ vinyl-fs@0.3.14
    └─┬ glob-watcher@0.0.6
      └─┬ gaze@0.5.2
        └─┬ globule@0.1.0
          └── lodash@1.0.2
```
The issue in this tree is actually gulp itself, which for some reason has fallen behind on vinyl-fs' development. The latest vinyl-fs doesn't even have a direct dependency on glob-watcher

**Comment by jorrit** [comment]
Gulp 4 will fix that.

**Comment by peterjhart** [comment]
I doubt Gulp 4 is never coming out :\

**Comment by jdalton** [comment]
\@phated

**Comment by Eruant** [comment]
Any update on this?

**Comment by adambuczynski** [comment]
+1

**Comment by mdvorscak** [comment]
```
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
├─┬ gulp@3.9.1 
...
│ └─┬ vinyl-fs@0.3.14 
│   ├─┬ glob-watcher@0.0.6 
│   │ └─┬ gaze@0.5.2 
│   │   └─┬ globule@0.1.0 
│   │     ├─┬ glob@3.1.21 
│   │     │ ├── graceful-fs@1.2.3 
│   │     │ └── inherits@1.0.2 
│   │     ├── lodash@1.0.2 
```
+1

**Comment by shama** [comment]
Fixed on b54c578e5fee74e9e9fcca23efed4c296fece230
